chromadb/segment/impl/vector/local_persistent_hnsw.py

```diff
@@ -52,29 +49,17 @@ class PersistentData:
     def __init__(
         self,
         dimensionality: Optional[int],
-        total_elements_added: int,
-        total_elements_updated: int,
-        total_invalid_operations: int,
         max_seq_id: int,
         id_to_label: Dict[str, int],
         label_to_id: Dict[int, str],
         id_to_seq_id: Dict[str, SeqId],
     ):
         self.dimensionality = dimensionality
-        self.total_elements_added = total_elements_added
-        self.total_elements_updated = total_elements_updated
-        self.total_invalid_operations = total_invalid_operations
         self.max_seq_id = max_seq_id
         self.id_to_label = id_to_label
         self.label_to_id = label_to_id
         self.id_to_seq_id = id_to_seq_id
 
-    def __setstate__(self, state: Any) -> None:
-        # Fields were added after the initial implementation
-        self.total_elements_updated = 0
-        self.total_invalid_operations = 0
-        self.__dict__.update(state)
```

Comment:
I tested with pickle and it seems to be fine if you remove a field from a class def